src/index.html

```diff
@@ -1,13 +1,363 @@
+
+
+            <img
+              src="images/crumbs.png"
+              alt="Crumbs image"
+              class="about-us__img"
+            >
+
+
+            <div class="about-us__facts">
+              <p class="about-us__number">2015</p>
+              <p class="about-us__number-details">Foundation year</p>
+            </div>
+
+            <div class="about-us__facts">
+              <p class="about-us__number">23</p>
+              <p class="about-us__number-details">Stores </p>
+            </div>
+
+            <div class="about-us__facts">
+              <p class="about-us__number">155</p>
+              <p class="about-us__number-details">Companies </p>
+            </div>
+
+            <div class="about-us__facts">
+              <p class="about-us__number">10K+</p>
+              <p class="about-us__number-details">Clients </p>
+            </div>
+          </section>
+        </div>
+      </div>
+    </main>
+    <footer class="footer" id="contacts">
+      <div class="container">
+        <div class="footer__content">
+          <div class="footer__menu-wrapper">
+            <a href="#" class="footer__logo-link">
+              <img
+                src="images/logo-2x-white.png"
+                alt="Cooking Bakery logo"
+                class="footer__logo"
+              >
+            </a>
+            <nav class="nav footer__nav">
+              <ul class="nav__list footer__list">
+                <li class="nav__item footer__item">
+                  Menu
+                </li>
+                <li class="nav__item">
+                  <a class="nav__link" href="#products">Products</a>
+                </li>
+                <li class="nav__item">
+                  <a class="nav__link" href="#about-us">About us</a>
+                </li>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Blog</a>
+                </li>
+                <li class="nav__item">
+                  <a class="nav__link" href="#">Terms & Privacy</a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+          <section class="contact-us" id="contact-us">
+            <div class="footer__contact-wrapper">
+              <h2 class="
+              section-title
+              section-title--align--left
+              section-title--contact
+              ">Contact us</h2>
+              <div class="footer__list-wrapper">
+                <ul class="nav__list footer__list">
+                  <li class="nav__item footer__item">
+                    Call us
+                  </li>
+                  <li class="nav__item">
+                    <a class="nav__link" href="tel:(629) 555-0129 ">(629) 555-0129</a>
```

Comment:
Need to fix format phone everywhere only in `href`
```suggestion
                    <a class="nav__link" href="tel:6295550129">(629) 555-0129</a>
```